packages/angular_devkit/build_optimizer/package.json

```diff
@@ -8,6 +8,7 @@
     "build-optimizer": "./src/build-optimizer/cli.js"
   },
   "dependencies": {
+    "@angular-devkit/core": "^7.0.2",
```

Comment:
I don't think this should be fixed at 7.0.2 - do we really need this line added?

Reply:
yeah, it's not needed. deleted this line.